spinalcordtoolbox/deepseg/monai.py

```diff
@@ -112,7 +112,7 @@ def create_nnunet_from_plans(path_model):
         model.apply(init_last_bn_before_add_to_0)
 
     # this loop only takes about 0.2s on average on a CPU
-    chkp_path = os.path.join(path_model, "best_model_loss.ckpt")
+    chkp_path = os.path.join(path_model, "model", "best_model.ckpt")
```

Comment:
Probably not something we need to change now, but maybe something to consider when we add more MONAI models in the future.

Reply:
Thanks for the quick review, Joshua! Yeah, maybe this could be safe option for the future. At least for future contrast-agnostic model, I will not change this again !

Reply:
I think this is a good change! This will make sure that if someone updates their code but forgets to re-run `sct_download_data`, they won't get an error.

Though, I am thinking about which error would be thrown if the model is missing. Before, a `FileNotFoundError` would be thrown, which is very intuitive. But, with globbing + `[0]`, an `IndexError` would be thrown, which is a bit less intuitive.)

So, perhaps something like this?

```python
    chkp_paths = glob.glob(os.path.join(path_model, '**', '*.ckpt'), recursive=True)
    if not chkp_paths:
        raise FileNotFoundError(f"Could not find .ckpt file in {path_model}")
    chkp_path = chkp_path[0]
```

Reply:
amazing! 

> I think this is a good change! This will make sure that if someone updates their code but forgets to re-run sct_download_data, they won't get an error.

definitely didn't think of this

I kind of assumed our releases will always have the model (:p) so added a `[0]` there! but you're right, your suggestion is more failsafe!

Reply:
updated here! 302871f6